toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -162,13 +158,6 @@ func createImageCustomizerParameters(buildDir string,
 	ic.outputImageDir = filepath.Dir(ic.outputImageFile)
 	ic.outputPXEArtifactsDir = outputPXEArtifactsDir
 
-	if ic.outputImageFormat != "" && !ic.outputIsIso {
```

Comment:
Why was this validation removed? I think this exists so that we fail early in the run instead of at the very end.

Reply:
The `CustomizerImage` function is called directly by the tests. So, it can be handy to include/duplicate all the verification checks to make writing tests easier.